examples/dt/uni07eta/edpm/values.yaml

```diff
@@ -132,4 +135,4 @@ data:
       - ovn
       - neutron-metadata
       - libvirt
-      - nova
+      - nova-custom
```

Comment:
This works but couple of weeks ago we added an optional config map to the default DataPlaneService/nova as well https://github.com/openstack-k8s-operators/openstack-operator/blob/252261c1584a471a5cb83b249c64cef230fd918c/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml#L12  

So this PR can be simplified to only add content to CM/nova-extra-config